json/src/test/java/com/basistech/rosette/dm/json/plain/JsonTest.java

```diff
@@ -491,7 +502,16 @@ public void oneWithEverything() {
         crListBuilder.add(topicResult);
         builder.topicResults(crListBuilder.build());
 
+        Embeddings.Builder embeddingsBuilder = new Embeddings.Builder();
+        EmbeddingCollection.Builder embeddingCollectionBuilder = new EmbeddingCollection.Builder();
+        embeddingCollectionBuilder.put(0, new float[]{0, 1});
+        embeddingsBuilder.put(Embeddings.TEXT, embeddingCollectionBuilder.build());
+        embeddings = embeddingsBuilder.build();
+        builder.embeddings(embeddingsBuilder.build());
+
```

Comment:
I don't have a great idea here. If you prefer a convenience accessor I'll add it.